ceph.spec.in

```diff
@@ -797,18 +800,42 @@ export CPPFLAGS="$java_inc"
 export CFLAGS="$RPM_OPT_FLAGS"
 export CXXFLAGS="$RPM_OPT_FLAGS"
 
-env | sort
-
+# Parallel build settings ...
+# unlimit _smp_mflags in system macro
+%global _smp_ncpus_max 0
+CEPH_PARALLEL_BUILD="%{?_smp_mflags}"
```

Comment:
why not just set `_smp_mflags`? we are practically overriding it here. also the  `CEPH_PARALLEL_BUILD` is confusing, might want to use something like `CEPH_MFLAGS_JOBS`.

Reply:
@tchaikov Good question. I tried, but failed, to create the equivalent of `CEPH_MFLAGS_JOBS="-j$lo_jobs"` as an RPM macro definition. That doesn't mean there is no way to get an expanded shell variable into a macro definition - just that I don't know how.

Reply:
I changed `CEPH_PARALLEL_BUILD` to `CEPH_MFLAGS_JOBS`.